src/main/kotlin/anttracker/issues/Issue.kt

```diff
@@ -38,6 +48,10 @@ data class IssueInformation(
     val priority: Priority,
 )
 
+/** ---
+ * This data class represents the valid values an issue id can take on,
+ * being between 1-99.
```

Comment:
Where does 99 come from?

Reply:
I was basing it off our BNF diagram requirements. 
<img width="684" alt="image" src="https://github.com/user-attachments/assets/8175b557-8146-4742-b4b0-4f872800f499">